numpy/core/tests/test_regression.py

```diff
@@ -436,6 +436,32 @@ def __getitem__(self, key):
 
         assert_raises(KeyError, np.lexsort, BuggySequence())
 
+    def test_lexsort_zerolen_custom_strides(self):
+        # Ticket #14228
+        xs = np.array([], dtype='i8')
+        assert xs.strides == (8,)
+        assert np.lexsort((xs,)).shape[0] == 0 # Works
+
+        xs.strides = (16,)
+        assert np.lexsort((xs,)).shape[0] == 0 # Was: MemoryError
```

Comment:
NIT: extra space before the `#` (doesn't matter much).

Reply:
I'm honestly not sure what you are suggesting here. You really prefer to see `assert np.lexsort((xs,)).shape[0] == 0# Was: MemoryError`? This looks weird to me and doesn't match the style of other comments in this file.

Reply:
Doesn't matter much. I meant two spaces, I believe that is the suggestion in PEP 8

Reply:
PEP 8: https://www.python.org/dev/peps/pep-0008/#inline-comments:  "Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space."

Reply:
Thanks for explaining.